### ISSUE TITLE
Success message 'You’ve successfully ...' is displayed if user submitted subscribed email in newsletter field

### ISSUE THREAD

**Issue by aleksandrakorolova** [scandipwa/scandipwa#1784]
**Description**:
Success message 'You’ve successfully subscribed to our newsletter' is displayed when entering subscribed email

**Expected behavior**:
Error message 'This email address is already subscribed' is displayed

**To Reproduce**:
1. Open website
2. Scroll down to footer
3. Enter subscribed email
4. Tap Letter icon

**Actual result:**:
Success message 'You’ve successfully subscribed to our newsletter' is displayed

**Comment by niklavskatlaps** [comment]
Just investigated this issue and it's actually working as expected.

When a user submits an email in the newsletter field, the status is changed to `STATUS_NOT_ACTIVE` - it means that we are waiting for a user to click the activation link in the newsletter subscription confirmation email. 
![Screenshot at 15-53-02](https://user-images.githubusercontent.com/22444018/103413101-cf416a80-4b80-11eb-8b0a-d89d95c29040.png)

Once the confirmation is done, the status is changed to `STATUS_SUBSCRIBED`. 
![Screenshot at 15-50-13](https://user-images.githubusercontent.com/22444018/103413103-d0729780-4b80-11eb-8101-a719e39207c0.png)

And if we try to submit the same email again, we get the expected error message.
![Screenshot at 15-59-53](https://user-images.githubusercontent.com/22444018/103413208-3c550000-4b81-11eb-829e-09e8d11a50a5.png)

I manually confirmed my subscription in the BE. **I couldn't find an email template for subscription confirmation so I guess it needs to be created** but other than that this is not a bug and I think that this issue can be closed.

**Comment by aleksandrakorolova** [comment]
Thank you for this information!
Closed this task. Separate task was created #2024
